packages/suite/src/utils/suite/__tests__/env.test.ts

```diff
@@ -1,7 +1,14 @@
 import * as env from '@suite-utils/env';
 import * as fixtures from '../__fixtures__/env';
 
-describe('isMacOs', () => {
+// TODO: this tests should be reworked due to mocking problem because of compilation.
```

Comment:
We should probably create an issue for it at the time of merging those skips in develop.

Reply:
https://github.com/trezor/trezor-suite/issues/4993